src/CMakeLists.txt

```diff
@@ -763,7 +783,7 @@ ENDIF (WANT_INTERNAL_LOKI)
 # prevent opening the cmd.exe window
 IF (WIN32)
   LIST(APPEND TORA_LIBS ${WINDOWS_LIB})
-  LIST(APPEND TORA_LIBS "ws2_32.lib")
+  #LIST(APPEND TORA_LIBS "ws2_32.lib")
```

Comment:
ok lets remove it. This probably was required on Win XP when using MSVC 2010. This library was merged into some other Windows system lib on newer Windows releases.

PS: does QT5 build work for you? When I use precompiled Qt5 libs(5.5.0) from qi.io, the application randomly crashes for me. Even more MSVC 2013 fails to load debug symbols (.pdb) files telling me those are no compatible with .dlls used.  When I compiled Qt5 my own Qt5 libs it worked better, but I think it is not the right way.